### ISSUE TITLE
App not rendering on local dev setup

### ISSUE THREAD

**Issue by pelhage** [serverless/site#12]
I haven't been able to get up and running with a local copy of serverless.com just yet. Currently seeing `...` and an error message in the console.

I have updated my .env.sample to .env and included my Auth0 credentials but no dice

![screenshot 2016-09-15 19 53 21](https://cloud.githubusercontent.com/assets/4411839/18574122/b6541936-7b7e-11e6-945e-62c390f685d9.png)

**Comment by pelhage** [comment]
FYI, I have not set up serverless myself, so let me know if I have to go through the serverless guide and set up an account with AWS before being able to edit docs

**Comment by DavidWells** [comment]
Hey @pelhage you the man!

Try to rename https://github.com/serverless/site/blob/master/.env.sample file from `.env.sample` to `.env`

Webpack needs some credentials to use to run throught the build

**Comment by pelhage** [comment]
@DavidWells I did rename my `.env.sample` to `.env`. Could it have anything to do with my Auth0 account being free with limited features? https://auth0.com/pricing#full-features

Or maybe some extra configuration that I needed to set up with Auth0? (All I did was create an account and copy paste the credentials)

Here's my error related to `phenomic build` when running `npm run build`, related to highlight.js & that arduino.js error in console previously mentioned:

![screenshot 2016-09-21 14 51 56](https://cloud.githubusercontent.com/assets/4411839/18730543/c6944938-800b-11e6-9d15-aea74a7a9b33.png)
